promptsource/templates/khalidalt/HuffPost/templates.yaml

```diff
@@ -0,0 +1,96 @@
+dataset: khalidalt/HuffPost
+templates:
+  19d2449b-1a52-4079-aaec-301517ee6ee1: !Template
+    answer_choices: POLITICS|||  WELLNESS|||  ENTERTAINMENT|||  TRAVEL|||  STYLE &
+      BEAUTY|||  PARENTING|||  HEALTHY LIVING|||  QUEER VOICES|||  FOOD & DRINK|||  BUSINESS|||  COMEDY|||  SPORTS|||  BLACK
+      VOICES|||  HOME & LIVING|||  PARENTS|||  THE WORLDPOST|||  WEDDINGS|||  WOMEN|||  IMPACT|||  DIVORCE|||  CRIME|||  MEDIA|||  WEIRD
+      NEWS|||  GREEN|||  WORLDPOST|||  RELIGION|||  STYLE|||  SCIENCE|||  WORLD NEWS|||  TASTE|||  TECH|||  MONEY|||  ARTS|||  FIFTY|||  GOOD
+      NEWS|||  ARTS & CULTURE|||  ENVIRONMENT|||  COLLEGE|||  LATINO VOICES|||  CULTURE
+      & ARTS|||  EDUCATION
+    id: 19d2449b-1a52-4079-aaec-301517ee6ee1
+    jinja: 'What can you infer from the headline and short description given about
```

Comment:
This is ungrammatical. How about "Given these headline and description, what category can you infer this piece of news belongs to?"

Reply:
> Thanks for the PR! A few issues left:
> 
>     1. None of your prompts have the answer choices in the input, so (a) models or even humans will never know to answer "impact" or "divorce" as a valid news category. (b) This may be mitigated by rank classification of the answer choices, but it's far from guaranteed. At the very lest, you need to set the `choices_in_prompts` accordingly.
> 
>     2. Please lowercase or title case your answer choices. Most models are case-sensitive now and are a priori unlikely to generate all caps vocabulary.
> 
>     3. The classes are imbalanced, so you probably don't really want to report naive accuracy. Fine for now, but please pay attention in you evaluation.
> 
>     4. What is the file change of `promptsource/templates/tydiqa/primary_task/.templates.yaml.swp`?

1- I can add answer choices as list in the input as follow:

`Given the following news headline and short description: 
{{headline}}
{{short_description}}
What is a correct category class from the following list {{answer_choices}} ? ||| {{answer_choices[label]}}`

However, It may seem unnatural to use this style with humans. 

2- Fixed. I titled case all answer choices. 

3- OK

4- Fixed. It was a mistake and I removed the file.